src/utils/ws.ts

```diff
@@ -0,0 +1,14 @@
+import { isNodeJS } from './environment'
+
+// RN exposes global WebSocket
+let WebSocket = global.WebSocket
+
+// NodeJS doesn't have WebSocket by default
+if (!WebSocket && isNodeJS()) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const nodeWebSocket = require('ws')
```

Comment:
I think this `ws` package will be part of the React Native bundle anyway. Not saying it's bad, because we would like to address this by monorepo. Just thought it's worth mentioning.

Reply:
React Native has a built-in WebSocket client implementation. Why would we need the `ws` library?

Reply:
Oh, sorry, I meant that this logic will happen only in runtime, so when you build RN app then the output JS bundle may contain also the `ws` package which is not needed as you said.

Reply:
But do you see where the problem is here, right? :)

Reply:
Yeah I see the problem. I'm just not that set on fixing this before we move to a monorepo :)

I already have the structure, the most important thing is determining how to separate the packages.

Reply:
Good good, of course, I didn't mean it as a blocker 👍

Reply:
See #270 for discussion on monorepo